services/tenant-ui/frontend/src/components/contacts/acceptInvitation/AcceptInviteForm.vue

```diff
@@ -96,14 +94,15 @@ const handleSubmit = async (isFormValid: boolean) => {
   }
 
   try {
-    // await contactsStore.acceptInvitation(
-    //   formFields.inviteUrl,
-    //   formFields.alias
-    // );
-    emit('success');
-    // close up on success
-    emit('closed');
-    toast.info('Invitation Accepted');
+    const inviteParam = paramFromUrlString(
+      formFields.inviteUrl,
+      'c_i'
+    ) as string;
+    if (!inviteParam) {
+      throw Error('Invalid format for invitation URL');
+    }
+    invitationString.value = window.atob(inviteParam);
```

Comment:
not sure if it is worth dealing with at this time but `http:/hey.you?c_i=blah` passes but then blows up because invalid json, kind of buggers up the table load.

Reply:
Yes, good call.
Validating the JSON using a the form validation to block submission should work (and let the user see what the decoded URL ends up being)
![image](https://user-images.githubusercontent.com/17445138/220533578-3364e627-f8f5-4cb4-99a0-94c5056ead0b.png)